backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/OAuthService.java

```diff
@@ -3,16 +3,25 @@
 import com.canvas.exceptions.CanvasAPIException;
 import com.fasterxml.jackson.databind.JsonNode;
 import okhttp3.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
 
 /**
  * This class handles OAuth requests for generation, validation of access tokens
  */
 public class OAuthService {
 
     private final CanvasClientService canvasClientService;
+    private String clientSecret;
+    private String clientId;
     //TODO: Add and fetch values from config
-    private final String CLIENT_ID = "10000000000002";
-    private final String CLIENT_SECRET = "xppoeCamrsQVsEdZi5g6ykNE1qEJort6Zdn8ojGTcTBzRO15fR0AEdeeCtqcLymI";
+
+    @Autowired
+    public OAuthService(Environment env, CanvasClientService canvasClientService) {
+        this.clientId = env.getProperty("canvas.clientId");
+        this.clientSecret = env.getProperty("canvas.clientSecret");
+        this.canvasClientService = canvasClientService;
+    }
     public OAuthService(CanvasClientService canvasClientService){
```

Comment:
This is showing as never used in my IDE. What's the purpose of this method?

Reply:
Removed it.